lib/src/html_generator.dart

```diff
@@ -41,10 +41,10 @@ class Templates {
   TemplateRenderer topLevelPropertyTemplate;
   TemplateRenderer typeDefTemplate;
 
-  final Map<String, String> _partialTemplates = {};
+  final Map<String, String> _partialTemplates = <String, String>{};
```

Comment:
Generics are reified. This allows you to get runtime exceptions in checked mode if the wrong kind of item is added.